lib/features/push_notification/data/repository/fcm_repository_impl.dart

```diff
@@ -211,12 +217,22 @@ class FCMRepositoryImpl extends FCMRepository {
     }
 
     if (emailChangeResponse?.created?.isNotEmpty == true) {
-      return emailChangeResponse!.created!.listEmailIds;
+      return _validateMaximumNewEmailsRetrieved(emailChangeResponse!.created!);
```

Comment:
Because we only care about the last 5, why need to put it in `while` loop? How about to set the limit to 5, and sort directly on query?

Reply:
We cannot be sure that `Email/change` returns in the most recent chronological order. So we need to get all of them then sort them by time and limit the first 5 emails.

Reply:
ok, so please remove while loop.

Reply:
done